Sunrise/de.lproj/Localizable.strings

```diff
@@ -175,4 +175,16 @@
 /* Get Directions */
 "Get Directions" = "Anfahrt";
 
+/* Discounts */
+"Discounts" = "Rabatt";
+
+/* Add discount code message */
+"Add your discount code bellow:" = "Fügen Sie Ihren Rabatt Code unten hinzu:";
+
+/* Discount */
+"Discount" = "Rabatt";
+
+/* No discounts */
+"No discounts - please redeem a code." = "Keine Rabatte - bitte erlöschen Sie einen Code.";
```

Comment:
Good point 👍 

Since the UX will likely change with the new design, I'd try to fix this by removing the ℹ️ icon when there are no discount codes present. Does that sound like an OK approach for the time being?